src/persistence/settingsserializer.cpp

```diff
@@ -412,11 +411,9 @@ void SettingsSerializer::readIni()
         // Add all keys
         if (!s.group().isEmpty())
             beginGroup(s.group());
+
         for (QString k : s.childKeys())
-        {
             setValue(k, s.value(k));
-            //qDebug() << "Read key "<<k<<" in group "<<group<<":\""<<groups[group]<<"\"";
-        }
```

Comment:
I agree, that brackets should be everywhere (for `if / while / for` with one line too). I will happy if maintainers change this rule :)